### ISSUE TITLE
Wrong log level

### ISSUE THREAD

**Issue by flamhero** [agile-lab-dev/wasp#6]
Each time I start the application, I can see these lines in the log:
```
ERROR c.v.a.w.c.s.l.SparkConsumersStreamingNodeLauncher$: Starting collaborator
ERROR c.v.a.w.c.s.l.SparkConsumersStreamingNodeLauncher$: Collaborator started
```
Unless I am mistaken, they are coming from this code: [link](https://github.com/agile-lab-dev/wasp/blob/a1dc468cfbcabf3639d6f015d99fbc4f91a5ab8f/consumers-spark/src/main/scala/it/agilelab/bigdata/wasp/consumers/spark/launcher/SparkConsumersStreamingNodeLauncher.scala#L70-L72)

Obviously, the log level should be INFO instead of ERROR.

**Comment by andrea-rockt** [comment]
yep you are right, anyway this is harmless (was done only to raise visibility of that log line and was left as it is)

**Comment by tmnd1991** [comment]
You're more than welcome to open a PR :)
